manim/cli/cfg/group.py

```diff
@@ -271,13 +272,15 @@ def export(ctx, directory):
     else:
         proceed = True
     if proceed:
-        if not os.path.isdir(directory):
+        if not Path(directory).is_dir():
             console.print(f"Creating folder: {directory}.", style="red bold")
-            os.mkdir(directory)
-        with open(os.path.join(directory, "manim.cfg"), "w") as outpath:
+            Path(directory).mkdir(parents=True)
+        with open(
+            Path(directory).joinpath("manim.cfg"), "w"
+        ) as outpath:  # this is not used?
             ctx.invoke(write)
-            from_path = os.path.join(os.getcwd(), "manim.cfg")
-            to_path = os.path.join(directory, "manim.cfg")
+            from_path = Path(Path.cwd()).joinpath("manim.cfg")
```

Comment:
```suggestion
            from_path = Path.cwd() / "manim.cfg"
```
Path.cwd() returns a Path object so casting is not required also you can use the `/` operator.

Reply:
Fixed!